src/reactor/fns.js

```diff
@@ -0,0 +1,444 @@
+import Immutable from 'immutable'
+import logging from '../logging'
+import { isImmutableValue } from '../immutable-helpers'
+import { toImmutable } from '../immutable-helpers'
+import { fromKeyPath, getStoreDeps, getComputeFn, getDeps, isGetter } from '../getter'
+import { isEqual, isKeyPath } from '../key-path'
+import { each } from '../utils'
+
+/**
+ * Immutable Types
+ */
+var EvaluateResult = Immutable.Record({ result: null, reactorState: null})
```

Comment:
Good catch!

I will take a pass through all of the `src` code and remove vars and other es5 constructs that have better es6 replacements.